zigpy/zdo/types.py

```diff
@@ -647,6 +657,13 @@ class ZDOCmd(t.enum_factory(_CommandID)):
     # Network Management Server Services Responses
     ZDOCmd.Mgmt_Lqi_rsp: (STATUS, ("Neighbors", t.Optional(Neighbors))),
     ZDOCmd.Mgmt_Rtg_rsp: (STATUS, ("Routes", t.Optional(Routes))),
+    ZDOCmd.Mgmt_Bind_rsp: (
+        STATUS,
+        ("BindingTableEntries", t.uint8_t),
+        ("StartIndex", t.uint8_t),
+        ("BindingTableListCount", t.uint8_t),
+        ("BindingTableList", t.List[Binding]),
```

Comment:
Thanks!

Since the size of the table precedes its length, could you merge:

```Python
        ("BindingTableListCount", t.uint8_t),
        ("BindingTableList", t.List[Binding]),
```

Into:

```Python
        ("BindingTableList", t.LVList[Binding]),
```